ui/app/templates/rule_scheduled_change_modal.html

```diff
@@ -205,10 +205,29 @@ <h4 ng-show= "sc.change_type !== 'delete'">Rule Details</h4>
         </div>
       </div>
       <div class="col-md-6" ng-show= "sc.change_type !== 'delete'">
+        <div class="form-group" ng-class="{'has-error': errors.mig64}">
```

Comment:
BTW, I noticed that mig64 has different position in Edit Rule and Schedule Change dialogs, which turns out to be because Alias isn't in the latter. Is that deliberate ?